src/getHumanAge.test.js

```diff
@@ -7,4 +7,44 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('return listed years', () => {
```

Comment:
```suggestion
  test('return listed years', () => {
```

it's not the correct name for the test. We need to write like 'should return something when something', because when you test fails you will see this `return listed years - fail` and how will you know what happened

Reply:
fixed and updated